plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2954,6 +2954,17 @@ public void testSplitPruningForFilterOnNonPartitionColumn(DataMappingTestSetup t
         }
     }
 
+    @Test
+    public void testGetTableProperties()
+    {
+        assertUpdate("CREATE TABLE test_iceberg_get_table_props (x BIGINT) WITH (format='PARQUET')");
+        MaterializedResult result = computeActual("SELECT * FROM \"test_iceberg_get_table_props$properties\"");
+        assertEquals(result.getRowCount(), 1);
+        assertEquals(result.getMaterializedRows().get(0).getField(0), "write.format.default");
+        assertEquals(result.getMaterializedRows().get(0).getField(1), "PARQUET");
+        dropTable("test_iceberg_get_table_props");
```

Comment:
i recommend doing the drop of the table in a `try`/`finally` clause.

Reply:
i recommend not using `finally`. it doesn't add value (unless test is run on shared resources, which is not the case), and clutters the test code.